.travis.yml

```diff
@@ -17,23 +18,30 @@ matrix:
         - cargo clippy --all -- -D warnings
         - cargo fmt -- --check
     - language: generic
-      dist: xenial
-      sudo: required
+      name: "dƒusion e2e Tests"
       cache: 
         directories:
-          - $HOME/.nvm/versions/node/v8.12.0/lib/node_modules
           - dex-contracts/node_modules/
       install:
-        - git submodule update --init
-        - npm install -C dex-contracts
-        - cd dex-contracts && npx truffle compile && cd -
-        - docker-compose up -d driver graph-listener | while read line ; do echo "$(date)| $line"; done; # prints timestamps for profiling
-        - cd dex-contracts && npx wait-port -t 30000 8545 && npx truffle migrate && cd -
+        - ./test/setup_contracts.sh
```

Comment:
Why do you start ganache from a docker? 
Didn't we had that before and changed it, since running ganache "naitively" was faster?

Reply:
This has been like this for a while. 
I don't fully remember why this was better, but for one we can get better logging in docker. I also don't think it's much faster but am happy to be convinced otherwise (not subject to this PR though).

Reply:
I am in favour of having ganache as part of the container orchestration (since is it a part of the project). Leaving it out completely seems somewhat incomplete. Like the demo wouldn't run without it.